### PR TITLE
feat: doc-tools migrate to windicss

### PR DESCRIPTION
**Pull request by sanyuan0704**
# PR Details

之前的 Unocss 原子化方案与 Webpack5 持久化缓存冲突，因此考虑到构建性能问题，打算废弃 Unocss，同时考虑到改造成本，最终迁移到 WindiCSS 方案。

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by chenjiahan** [comment]
windicss 看起来已经停止维护了，可能直接用 tailwindcss 更合适

**Comment by sanyuan0704** [comment]
tailwind 迁移成本太高了，不支持 attributify 的写法。windicss 目前我看功能都够用

File: packages/cli/doc-core/package.json [hunk 1]
```diff
@@ -94,6 +94,8 @@
     "unist-util-visit-children": "^2.0.1",
     "unocss": "^0.46.5",
```
Comment (chenjiahan) [inline packages/cli/doc-core/package.json:95]: unocss 依赖是不是可以删了

**Comment by chenjiahan** [comment]
> tailwind 迁移成本太高了，不支持 attributify 的写法。windicss 目前我看功能都够用

okk，那先用着，问题不大

File: packages/cli/doc-core/package.json (hunk 1 above)
Comment (sanyuan0704) [reply packages/cli/doc-core/package.json:95]: 是的，我清理下

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
lock file 还需要更新一下哈

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**